_data/gaming.yml

```diff
@@ -203,6 +203,13 @@ websites:
     twitter: MojangSupport
     img: mojang.png
 
+  - name: Nexon America
```

Comment:
From what I see there's nothing different between the login systems for the US version or any other version.
```suggestion
  - name: Nexon
```

Reply:
I can't confirm for other regions, unfortunately. If using "Nexon" as a whole instead is fine, sure.